libraries/botbuilder-storage-mssql/src/schema.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * @module @botbuildercommunity/storage-mssql
+ */
+
+export interface MSSQLOptions {
+    dbuser: string;
```

Comment:
Is there no interface available from the package or typings? Is it an idea to keep the interface the same as the config object in the `node-mssql`. The naming + the encrypt possibility.

```
const config = {
    user: '...',
    password: '...',
    server: 'localhost', // You can use 'localhost\\instance' to connect to named instance
    database: '...',
 
    options: {
        encrypt: true // Use this if you're on Windows Azure
    }
}
```

Reply:
I can have the interface inherit from the mssql package config. I'm just adding two additional options.

Reply:
Changed with commit: a9a8e6dee11857a87bd82cca310fdfb18d928d57.